models/tesolin2023hint/meadata.yaml

```diff
@@ -0,0 +1,31 @@
+title: Heterogeneous wIreless Network onTology
+acronym: hint
```

Comment:
Is this acronym really lower case?

Reply:
That is how it was informed by the author. To be honest, I am not sure if intentionally or not. Do you think we should modify?